config/default_records/plate_purposes/007_cardinal_purposes.wip.yml

```diff
@@ -6,7 +6,3 @@ LCA Blood Array:
   type: PlatePurpose::Input
   stock_plate: true
   cherrypickable_target: false
-LCA Blood Bank:
-  type: PlatePurpose::Input
-  stock_plate: true
-  cherrypickable_target: false
```

Comment:
I notice that here we're getting rid of the 'LCA Blood Bank' as the input plate to the cell banking pipeline. Do you remember we talked about making it symmetrical - so that the submission worked the same way, starting on the LCA Blood Vac tube, for the library prep pipeline? Have you tested that path?